airbyte-integrations/connectors/source-retently/source_retently/source.py

```diff
@@ -66,12 +66,14 @@ def parse_response(
         stream_slice: Mapping[str, Any] = None,
         next_page_token: Mapping[str, Any] = None,
     ) -> Iterable[Mapping]:
-        data = response.json().get("data")
-        if data:
-            stream_data = data.get(self.json_path) if self.json_path else data
-            if stream_data:
-                for d in stream_data:
-                    yield d
+        data = response.json().get("data") or response.json()
```

Comment:
I think it would be better to be explicit depending on the stream, and therefore surcharge the `parse_response` when the expected response differs.

It should make potential errors easier to spot!